specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2020-03-15-preview/storagepool.json

```diff
@@ -904,6 +1058,45 @@
         }
       }
     },
+    "IscsiTargetUpdateProperties": {
+      "type": "object",
+      "description": "Properties of iSCSI Target request payload.",
+      "required": [
+        "tpgs"
+      ],
+      "properties": {
+        "provisioningState": {
+          "description": "State of the operation on the resource",
+          "type": "string",
+          "$ref": "#/definitions/ProvisioningState",
+          "readOnly": true
+        },
+        "status": {
+          "type": "string",
+          "enum": [
+            "Invalid",
+            "Unknown",
+            "Healthy",
+            "Unhealthy"
+          ],
+          "x-ms-enum": {
+            "name": "OperationalStatus",
+            "modelAsString": true
+          },
+          "readOnly": true,
+          "description": "Operational status of the Disk pool"
+        },
+        "tpgs": {
+          "type": "array",
+          "description": "list of iSCSI target portal groups",
+          "items": {
+            "$ref": "#/definitions/TargetPortalGroup"
```

Comment:
some properties in TargetPortalGroup can't be update, even some required properties of TargetPortalGroup can't be update (e.g. "authentication"). 
This definition makes the generated PSH code for update must not work.

We should include exact the properties can update here. There properties can't be update (or part of the property can't be update) should not be here.

Reply:
Fixed this -- thanks again for catching it!